### PR TITLE
Team page designed for the new website

### PR DESCRIPTION
**Pull request by abiduzz420**
##### Changes Made:
- Added `team.js` in `website/pages/en/`
- Added `team.css` in `website/static/css/`
- In `team.yml` the keys are changed from kebab case to camel case
- Moved `_data/team.yml` -> `website/data/team.yml`
- Moved `icons` directory: `_includes/icons` -> `website/static/img/icons`
- Included new team members

This PR solves issue #1523 

_Please let me know if I need to make any further modifications or changes._

### CODE REVIEW

**Comment by abiduzz420** [comment]
Changes can be viewed here: https://deploy-preview-1530--babel.netlify.com/team.html

**Comment by abiduzz420** [comment]
@hzoo I have taken note of the issue #1457. I like the idea of contributors sharing their first PR stories and memorable contributions to babel.
The current PR does not included these changes. I shall work on the mockups and present you some designs, by taking inspiration from other testimonial websites. 

What would be the upper limit of number of characters in the message ?

File: website/static/css/team.css [hunk 1]
```diff
@@ -0,0 +1,137 @@
+.page_header {
```
Comment (hzoo) [inline website/static/css/team.css]: Looks like there are similar styles from video.css https://github.com/babel/website/pull/1524/files#r164283741

maybe should move to another css like 'page' or to the babel.css file? Not sure the best way

File: website/static/css/team.css [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+}
+
+/* 
+  ##Device = Desktops
+  ##Screen = 1281px to higher resolution desktops
+*/
+
+@media (min-width: 1281px) {
+  .team_member {
+    width: 25%;
```
Comment (hzoo) [inline website/static/css/team.css]: Does flexbox itself not cover this kind of thing already?

File: website/static/css/team.css (hunk 1 above)
Comment (abiduzz420) [reply website/static/css/team.css]: Yeah. We can have it in babel.css file. I feel we should create `Header` component, and re-use them on both videos and team page.

File: website/static/css/team.css (hunk 2 above)
Comment (abiduzz420) [reply website/static/css/team.css]: Yes. You can achieve the same thing using flex-box as well. I have used media queries to set specific styles based on the screen width. Flexbox helps in layout settings without knowing explicit width, height values. Also, we can change flex layouts within media queries as well. Shall I do do the same using flexbox ?

Comment (abiduzz420) [reply website/static/css/team.css]: To be a bit more clear, flex box has more to do with how space is being used and media queries tells how content is being viewed.

**Comment by rajzshkr** [comment]
screen shot 2018-01-28 at 8 12 34 pm

Can you check this?

**Comment by abiduzz420** [comment]
Thanks @rajzshkr for looking into it. I didn't catch that. I have fixed the issue now. Please let me if we need to modify anything further :)

File: website/static/css/team.css (hunk 1 above)
Comment (abiduzz420) [reply website/static/css/team.css]: I have shifted common styles from video.css & team.css to babel.css

**Comment by hzoo** [comment]
I think we should can probably just move to `emotion` to get around the naming issues like we are doing in the repl?

**Comment by abiduzz420** [comment]
Yes I agree @hzoo . emotion would be the right choice.

**Pull request merged**